t/40-openqa-clone-job.t

```diff
@@ -3,25 +3,79 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 use Test::Most;
-use Test::Warnings ':report_warnings';
+use Test::Warnings qw(warning :report_warnings);
 use FindBin;
 use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
 use OpenQA::Test::TimeLimit '10';
 use OpenQA::Test::Utils qw(run_cmd test_cmd);
-
+use Test::MockModule;
+$ENV{OPENQA_CONFIG} = '';
 
 sub test_once {
     # Report failure at the callsite instead of the test function
     local $Test::Builder::Level = $Test::Builder::Level + 1;
-    # prevent all network access to stay local
-    test_cmd('OPENQA_CONFIG= unshare -r -n script/openqa-clone-job', @_);
+    test_cmd('script/openqa-clone-job', @_);
```

Comment:
not sure but I cant help and ask why do you change that. I assume that change the conditions that the test used to run upon